package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "platform-custom-scalars",
+  "version": "1.0.0",
```

Comment:
better change to some minor version until it's ready for the major

Reply:
Done.